packages/cashier/src/Components/Form/__tests__/account-transfer-form.spec.js

```diff
@@ -0,0 +1,472 @@
+        isMobile.mockReturnValue(true);
+        const props = mockProps();
+        props.account_limits = {
+            daily_transfers: {
+                mt5: {
+                    allowed: 1,
+                },
+                internal: {
+                    allowed: 1,
+                },
+            },
+        };
+
+        render(<AccountTransferForm {...props} is_dxtrade_allowed={false} />);
+
+        expect(
+            screen.getByText('You may transfer between your Deriv fiat, cryptocurrency, and DMT5 accounts.')
+        ).toBeInTheDocument();
+        expect(
+            screen.getByText(
+                'Each day, you can make up to 1 transfers between your Deriv accounts and up to 1 transfers between your Deriv and DMT5 accounts.'
+            )
+        ).toBeInTheDocument();
+    });
+
+    it('should show proper hint about mt5 remained transfers', () => {
+        isMobile.mockReturnValue(true);
+        const props = mockProps();
+        props.account_limits = {
+            daily_transfers: {
+                mt5: {
+                    available: 1,
+                },
+            },
+        };
+        props.selected_from.is_mt = true;
+        props.selected_to.is_mt = true;
+
+        render(<AccountTransferForm {...props} />);
+
+        expect(screen.getByText('You have 1 transfer remaining for today.')).toBeInTheDocument();
+    });
+
+    it('should show proper hint about dxtrade remained transfers', () => {
+        isMobile.mockReturnValue(true);
+        const props = mockProps();
+        props.account_limits = {
+            daily_transfers: {
+                dxtrade: {
+                    available: 1,
+                },
+            },
+        };
+        props.selected_from = { is_dxtrade: true, currency: 'USD' };
+        props.selected_to = { is_dxtrade: true, currency: 'USD' };
+
+        render(<AccountTransferForm {...props} />);
+
+        expect(screen.getByText('You have 1 transfer remaining for today.')).toBeInTheDocument();
+    });
+
+    it('should show proper hint about internal remained transfers', () => {
+        isMobile.mockReturnValue(true);
+        const props = mockProps();
+        props.account_limits = {
+            daily_transfers: {
+                internal: {
+                    available: 1,
+                },
+            },
+        };
+
+        render(<AccountTransferForm {...props} />);
+
+        expect(screen.getByText('You have 1 transfer remaining for today.')).toBeInTheDocument();
+    });
+
+    it("should show proper AccountTransferNote if 'is_dxtrade_allowed' prop is true", () => {
+        isMobile.mockReturnValue(true);
+        const props = mockProps();
+
+        render(<AccountTransferForm {...props} is_dxtrade_allowed />);
+        expect(
+            screen.getByText('You may transfer between your Deriv fiat, cryptocurrency, DMT5, and Deriv X accounts.')
+        ).toBeInTheDocument();
+    });
```

Comment:
I think that we already covered this assertion in "should show note with proper message if is_dxtrade_allowed" test case, so i think that we don't need this test case.